<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-P57AQ83" ignored="3" name="module1(failed)(failed)" tests="1" failures="1" timestamp="2020-08-29T22:51:40 IST" time="0.057" errors="0">
  <testcase name="testcase1" time="0.012" classname="TestNG1.s_1">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG1.s_1.testcase1(s_1.java:11)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase name="testcase3" time="0.0" classname="TestNG1.s_1">
    <ignored/>
  </testcase> <!-- testcase3 -->
  <testcase name="testcase2" time="0.0" classname="TestNG1.s_1">
    <ignored/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase4" time="0.0" classname="TestNG1.s_1">
    <ignored/>
  </testcase> <!-- testcase4 -->
</testsuite> <!-- module1(failed)(failed) -->
